Assets/Packages/com.studio23.ss2.authsystem.steam/Editor/SteamAuthGeneratorEditorWindow.cs

```diff
@@ -1,59 +1,32 @@
-using Studio23.SS2.AuthSystem.Steam.Core;
+using System.IO;
+using Studio23.SS2.AuthSystem.Steam.Data;
 using UnityEditor;
+using UnityEditor.Build;
 using UnityEngine;
 
 namespace Studio23.SS2.AuthSystem.Steam.Editor
 {
     public class SteamAuthGeneratorEditorWindow : EditorWindow
     {
-        private uint _steamAppId;
-
-        [MenuItem("Studio-23/Authentication/Provide Steam Auth Data")]
-        public static void ShowWindow()
-        {
-            GetWindow<SteamAuthGeneratorEditorWindow>("Steam Authenticator");
-        }
-
-        private void OnGUI()
+        [MenuItem("Studio-23/AuthSystem/Providers/Steam", false, 10)]
+        static void CreateDefaultProvider()
         {
-            // Display the title image
-            var titleImage = Resources.Load<Texture2D>("TitleImage");
-            GUILayout.Label(titleImage, GUILayout.MaxHeight(150f));
-
-            // Display a hint about where to get the Steam AppID
-            EditorGUILayout.HelpBox("To get your Steam AppID, visit: https://partner.steamgames.com/apps/create",
-                MessageType.Info);
-
-            EditorGUILayout.LabelField("Enter Steam App ID:");
-            _steamAppId = (uint)EditorGUILayout.IntField((int)_steamAppId);
+            SteamAuthProvider providerSettings = ScriptableObject.CreateInstance<SteamAuthProvider>();
+            
+            string resourceFolderPath = "Assets/Resources/AuthSystem/Providers";
```

Comment:
Should be statically accessible from authsystem package. changing provider path will require updating all packages otherwise.